From jeffm@suse.com  Mon Aug 10 16:32:45 2009
From: Jeff Mahoney <jeffm@suse.com>
Date: Tue, 04 Aug 2009 10:26:23 -0400
Subject: Staging: rtl8192su: add linux/vmalloc.h
To: Greg KH <gregkh@suse.com>
Message-ID: <20090804142710.377919293@suse.com>


Some arches don't provide vmalloc.h through the includes already included
in rtl8192su. This patch adds it.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/staging/rtl8192su/r8192U_core.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rtl8192su/r8192U_core.c
+++ b/drivers/staging/rtl8192su/r8192U_core.c
@@ -24,6 +24,7 @@
  * Jerry chuang <wlanfae@realtek.com>
  */
 
+#include <linux/vmalloc.h>
 
 #undef LOOP_TEST
 #undef DUMP_RX
